docs/Developers/Git_Basics.rst

```diff
@@ -453,3 +453,9 @@ use an interactive rebasing session. This opens an editor where you can enter
 commands (described below) for each commit to be rebased. These commands
 determine how individual commits will be transferred to the new base. You can
 also reorder the commit listing to change the order of the commits themselves.
+
+If you would like more information/commands, please use your favourite search
+engine to look for it. You could also read the
+`git documentation <https://git-scm.com/doc>`_ to learn more. Git is widely used
```

Comment:
Well, they added exactly what I suggested [here](https://github.com/coala/coala/pull/3980/files#r107846863). I find it beneficial to stress just how trivial it is to google git related stuff to keep people from asking them in the chatrooms.